tests/auto/baseline_collector.test.py

```
@@ -0,0 +1,454 @@
+#!/usr/bin/env python3
+"""
+Comprehensive tests for BaselineCollector class
+Tests all public methods with happy path and edge cases
+"""
+
+import pytest
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)